### PR TITLE
Jira: add parameters to query string i.o. request data.

### PR DESCRIPTION
**Pull request by FrankStappers**
Bugfix: pass parameters to query string i.o. request data.
Add additional parameters.

### CODE REVIEW

File: atlassian/jira.py [hunk 1]
```diff
@@ -1304,7 +1304,7 @@ def get_project_versions(self, key, expand=None):
             params['expand'] = expand
         return self.get('rest/api/2/project/{}/versions'.format(key), params=params)
 
-    def get_project_versions_paginated(self, key, start=None, limit=None, order_by=None, expand=None):
+    def get_project_versions_paginated(self, key, start=None, limit=None, order_by=None, expand=None, params=None):
```
Comment (gonchik) [inline atlassian/jira.py:1307]: Hi! Could you add the exact arguments, please?
https://docs.atlassian.com/software/jira/docs/api/REST/8.12.1/#api/2/project-getProjectVersionsPaginated

**Comment by gonchik** [comment]
Hi! 
@FrankStappers Can we wrap exact parameter? Otherwise it's confusing

**Comment by FrankStappers** [comment]
Hi @gonchik 

I added the two parameters "status" and "query". 
These parameters are mentioned in:
https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-project-versions/#api-rest-api-2-project-projectidorkey-version-get

**Comment by gonchik** [comment]
Thank you! :)

**Pull request merged**